cnf-certification-test/suite_test.go

```diff
@@ -264,3 +353,209 @@ func incorporateVersions(claimData *claim.Claim) {
 		ClaimFormat:  ClaimFormatVersion,
 	}
 }
+
+var upgrader = websocket.Upgrader{
+	CheckOrigin: func(r *http.Request) bool {
+		return true
+	},
+}
+
+// Define an HTTP handler that triggers Ginkgo tests
+func runHandler(w http.ResponseWriter, r *http.Request) {
+	// Run Ginkgo tests
+	var requestData RequestData
+	//var responseData ResponseData
+	// Parse JSON data from the request body
+	err := json.NewDecoder(r.Body).Decode(&requestData)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
```

Comment:
use a string to store the log file in memory, using :
in env: 
var buf *bytes.Buffer
var aString string

Initialize with:
	buf = bytes.NewBufferString(aString)

in this function:
	logrus.SetOutput(buf)